src/test/java/uk/gov/pay/ledger/transaction/resource/TransactionResourceIT.java

```diff
@@ -790,41 +901,11 @@ public void shouldReturnCSVHeadersInCorrectOrder() throws IOException {
         assertThat(header.get(17), is("Corporate Card Surcharge"));
         assertThat(header.get(18), is("Total Amount"));
         assertThat(header.get(19), is("Wallet Type"));
-        assertThat(header.get(20), is("Card Type"));
-    }
-
-    public void assertHealthyCsvResponse(InputStream csvStream, TransactionFixture transactionFixture) throws IOException {
-        List<CSVRecord> csvRecords = CSVParser.parse(csvStream, Charset.defaultCharset(), RFC4180.withFirstRecordAsHeader()).getRecords();
-
-        assertThat(csvRecords.size(), is(2));
-
-        CSVRecord paymentRecord = csvRecords.get(0);
-        assertPaymentTransactionDetails(paymentRecord, transactionFixture);
-        assertThat(paymentRecord.get("Amount"), is("1.23"));
-        assertThat(paymentRecord.get("State"), is("Error"));
-        assertThat(paymentRecord.get("Finished"), is("true"));
-        assertThat(paymentRecord.get("Error Code"), is("P0050"));
-        assertThat(paymentRecord.get("Error Message"), is("Payment provider returned an error"));
-        assertThat(paymentRecord.get("Date Created"), is("12 Mar 2018"));
-        assertThat(paymentRecord.get("Time Created"), is("16:25:01"));
-        assertThat(paymentRecord.get("Corporate Card Surcharge"), is("0.05"));
-        assertThat(paymentRecord.get("Total Amount"), is("1.23"));
-        assertThat(paymentRecord.get("test-key-1 (metadata)"), is("value1"));
-        assertThat(paymentRecord.get("Wallet Type"), is(""));
-
-        CSVRecord refundRecord = csvRecords.get(1);
-        assertPaymentTransactionDetails(refundRecord, transactionFixture);
-        assertThat(refundRecord.get("Amount"), is("-1.00"));
-        assertThat(refundRecord.get("State"), is("Refund error"));
-        assertThat(refundRecord.get("Finished"), is("true"));
-        assertThat(refundRecord.get("Error Code"), is("P0050"));
-        assertThat(refundRecord.get("Error Message"), is("Payment provider returned an error"));
-        assertThat(refundRecord.get("Date Created"), is("12 Mar 2018"));
-        assertThat(refundRecord.get("Time Created"), is("16:24:01"));
-        assertThat(refundRecord.get("Corporate Card Surcharge"), is("0.00"));
-        assertThat(refundRecord.get("Total Amount"), is("-1.00"));
-        assertThat(refundRecord.get("Wallet Type"), is(""));
-        assertThat(refundRecord.get("Issued By"), is("refund-by-user-email@example.org"));
+        assertThat(header.get(20), is("Fee"));
+        assertThat(header.get(21), is("Net"));
+        assertThat(header.get(22), is("Card Type"));
+        assertThat(header.get(23), is("MOTO"));
+        assertThat(header.get(24), is("a-metadata-key (metadata)"));
```

Comment:
This class is almost 1,000 lines long — might it be worth extracting the CSV tests into their own file? Some of the inserting of transactions that happens in each CSV test could probably be in to the `@Before` method then.

Reply:
Yeh good idea, it was difficult to find the relevant tests. I'll add another commit to move tests out.

Reply:
Split out the CSV and search tests. Didn't move any of the transaction setup to the `@Before` for CSV as the tests do need slightly different transactions